lib/barlix/svg.eex

```diff
@@ -1,9 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="<%= width %>px" height="<%=  height %>px" viewBox="0 0 <%= width %> <%= height %>" version="1.1" preserveAspectRatio="none" >
     <rect x="0" y="0" width="<%= width %>px" height="<%= height %>px" fill="#fff" />
-    <g fill="#000">
-        <%= for {x, y, bar_width, bar_height} <- bars do %>
-            <rect x="<%= x %>" y="<%= y %>" width="<%= bar_width %>px" height="<%= bar_height %>px" />
-        <% end %>
+    <g fill="#000"><%= for {x, y, bar_width, bar_height} <- bars do %>
+            <rect x="<%= x %>" y="<%= y %>" width="<%= bar_width %>px" height="<%= bar_height %>px" /><% end %>
     </g>
```

Comment:
is this just a formatting change? if so, could you revert it

Reply:
No, actually the test failed on my platform and this fixed it. When making changes, I wanted to have all tests pass before. This is `71108eb6efb9f`:

```
1) test print (Barlix.SVGTest)
     test/svg_test.exs:23
     Assertion with == failed
     code:  assert IO.iodata_to_binary(actual) == IO.iodata_to_binary(contents)
     left:  "<?xml version=\"1.0\" encoding=\"UTF-8\"?>\n<svg xmlns=\"http://www.w3.org/2000/svg\" xmlns:xlink=\"http://www.w3.org/1999/xlink\" width=\"123px\" height=\"120px\" viewBox=\"0 0 123 120\" version=\"1.1\" preserveAspectRatio=\"none\" >\n    <rect x=\"0\" y=\"0\" width=\"123px\" height=\"120px\" fill=\"#fff\" />\n    <g fill=\"#000\">\n            <rect x=\"10\" y=\"10\" width=\"1px\" height=\"100px\" />\n            <rect x=\"13\" y=\"10\" width=\"1px\" height=\"100px\" />\n            <rect x=\"15\" y=\"10\" width=\"2px\" height=\"100px\" />\n            <rect x=\"18\" y=\"10\" width=\"2px\" height=\"100px\" />\n            <rect x=\"21\" y=\"10\" width=\"1px\" height=\"100px\" />\n            <rect x=\"23\" y=\"10\" width=\"1px\" height=\"100px\" />\n            <rect x=\"25\" y=\"10\" width=\"2px\" height=\"100px\" />\n            <rect x=\"28\" y=\"10\" width=\"1px\" height=\"100px\" />\n            <rect x=\"31\" y=\"10\" width=\"1px\" height=\"100px\" />\n            <rect x=\"33\" y=\"10\" width=\"2px\" height=\"100px\" />\n            <rect x=\"36\" y=\"10\" width=\"2px\" height=\"100px\" />\n            <rect x=\"39\" y=\"10\" width=\"1px\" height=\"100px\" />\n            <rect x=\"41\" y=\"10\" width=\"1px\" height=\"100px\" />\n            <rect x=\"44\" y=\"10\" width=\"1px\" height=\"100px\" />\n            <rect x=\"46\" y=\"10\" width=\"2px\" height=\"100px\" />\n            <rect x=\"49\" y=\"10\" width=\"2px\" height=\"100px\" />\n            <rect x=\"52\" y=\"10\" width=\"1px\" height=\"100px\" />\n            <rect x=\"54\" y=\"10\" width=\"1px\" height=\"100px\" />\n            <rect x=\"56\" y=\"10\" width=\"2px\" height=\"100px\" />\n            <rect x=\"60\" y=\"10\" width=\"1px\" height=\"100px\" />\n            <rect x=\"62\" y=\"10\" width=\"1px\" height=\"100px\" />\n            <rect x=\"64\" y=\"10\" width=\"2px\" height=\"100px\" />\n            <rect x=\"67\" y=\"10\" width=\"1px\" height=\"100px\" />\n            <rect x=\"69\" y=\"10\" width=\"1px\" height=\"100px\" />\n            <rect x=\"72\" y=\"10\" width=\"2px\" height=\"100px\" />\n            <rect x=\"75\" y=\"10\" width=\"1px\" height=\"100px\" />\n            <rect x=\"77\" y=\"10\" width=\"2px\" height=\"100px\" />\n            <rect x=\"80\" y=\"10\" width=\"1px\" height=\"100px\" />\n            <rect x=\"83\" y=\"10\" width=\"2px\" height=\"100px\" />\n            <rect x=\"86\" y=\"10\" width=\"1px\" height=\"100px\" />\n            <rect x=\"88\" y=\"10\" width=\"1px\" height=\"100px\" />\n            <rect x=\"91\" y=\"10\" width=\"1px\" height=\"100px\" />\n            <rect x=\"93\" y=\"10\" width=\"2px\" height=\"100px\" />\n            <rect x=\"96\" y=\"10\" width=\"1px\" height=\"100px\" />\n            <rect x=\"98\" y=\"10\" width=\"2px\" height=\"100px\" />\n            <rect x=\"101\" y=\"10\" width=\"1px\" height=\"100px\" />\n            <rect x=\"104\" y=\"10\" width=\"1px\" height=\"100px\" />\n            <rect x=\"106\" y=\"10\" width=\"2px\" height=\"100px\" />\n            <rect x=\"109\" y=\"10\" width=\"2px\" height=\"100px\" />\n            <rect x=\"112\" y=\"10\" width=\"1px\" height=\"100px\" />\n    </g>\n</svg>\n"
     right: "<?xml version=\"1.0\" encoding=\"UTF-8\"?>\n<svg xmlns=\"http://www.w3.org/2000/svg\" xmlns:xlink=\"http://www.w3.org/1999/xlink\" width=\"123px\" height=\"120px\" viewBox=\"0 0 123 120\" version=\"1.1\" preserveAspectRatio=\"none\" >\n    <rect x=\"0\" y=\"0\" width=\"123px\" height=\"120px\" fill=\"#fff\" />\n    <g fill=\"#000\">\n\n            <rect x=\"10\" y=\"10\" width=\"1px\" height=\"100px\" />\n\n            <rect x=\"13\" y=\"10\" width=\"1px\" height=\"100px\" />\n\n            <rect x=\"15\" y=\"10\" width=\"2px\" height=\"100px\" />\n\n            <rect x=\"18\" y=\"10\" width=\"2px\" height=\"100px\" />\n\n            <rect x=\"21\" y=\"10\" width=\"1px\" height=\"100px\" />\n\n            <rect x=\"23\" y=\"10\" width=\"1px\" height=\"100px\" />\n\n            <rect x=\"25\" y=\"10\" width=\"2px\" height=\"100px\" />\n\n            <rect x=\"28\" y=\"10\" width=\"1px\" height=\"100px\" />\n\n            <rect x=\"31\" y=\"10\" width=\"1px\" height=\"100px\" />\n\n            <rect x=\"33\" y=\"10\" width=\"2px\" height=\"100px\" />\n\n            <rect x=\"36\" y=\"10\" width=\"2px\" height=\"100px\" />\n\n            <rect x=\"39\" y=\"10\" width=\"1px\" height=\"100px\" />\n\n            <rect x=\"41\" y=\"10\" width=\"1px\" height=\"100px\" />\n\n            <rect x=\"44\" y=\"10\" width=\"1px\" height=\"100px\" />\n\n            <rect x=\"46\" y=\"10\" width=\"2px\" height=\"100px\" />\n\n            <rect x=\"49\" y=\"10\" width=\"2px\" height=\"100px\" />\n\n            <rect x=\"52\" y=\"10\" width=\"1px\" height=\"100px\" />\n\n            <rect x=\"54\" y=\"10\" width=\"1px\" height=\"100px\" />\n\n            <rect x=\"56\" y=\"10\" width=\"2px\" height=\"100px\" />\n\n            <rect x=\"60\" y=\"10\" width=\"1px\" height=\"100px\" />\n\n            <rect x=\"62\" y=\"10\" width=\"1px\" height=\"100px\" />\n\n            <rect x=\"64\" y=\"10\" width=\"2px\" height=\"100px\" />\n\n            <rect x=\"67\" y=\"10\" width=\"1px\" height=\"100px\" />\n\n            <rect x=\"69\" y=\"10\" width=\"1px\" height=\"100px\" />\n\n            <rect x=\"72\" y=\"10\" width=\"2px\" height=\"100px\" />\n\n            <rect x=\"75\" y=\"10\" width=\"1px\" height=\"100px\" />\n\n            <rect x=\"77\" y=\"10\" width=\"2px\" height=\"100px\" />\n\n            <rect x=\"80\" y=\"10\" width=\"1px\" height=\"100px\" />\n\n            <rect x=\"83\" y=\"10\" width=\"2px\" height=\"100px\" />\n\n            <rect x=\"86\" y=\"10\" width=\"1px\" height=\"100px\" />\n\n            <rect x=\"88\" y=\"10\" width=\"1px\" height=\"100px\" />\n\n            <rect x=\"91\" y=\"10\" width=\"1px\" height=\"100px\" />\n\n            <rect x=\"93\" y=\"10\" width=\"2px\" height=\"100px\" />\n\n            <rect x=\"96\" y=\"10\" width=\"1px\" height=\"100px\" />\n\n            <rect x=\"98\" y=\"10\" width=\"2px\" height=\"100px\" />\n\n            <rect x=\"101\" y=\"10\" width=\"1px\" height=\"100px\" />\n\n            <rect x=\"104\" y=\"10\" width=\"1px\" height=\"100px\" />\n\n            <rect x=\"106\" y=\"10\" width=\"2px\" height=\"100px\" />\n\n            <rect x=\"109\" y=\"10\" width=\"2px\" height=\"100px\" />\n\n            <rect x=\"112\" y=\"10\" width=\"1px\" height=\"100px\" />\n\n    </g>\n</svg>\n"
     stacktrace:
       test/svg_test.exs:19: Barlix.SVGTest.assert_svg/3
       test/svg_test.exs:24: (test)
```